bin/cli.js

```diff
@@ -1,17 +1,171 @@
 #!/usr/bin/env node
 'use strict';
 
-const metasync = require('metasync');
+const program = require('commander');
+const karma = require('karma');
+const path = require('path');
+const fs = require('fs');
 
-const walk = require('../lib/walk');
+const parseFile = file => JSON.parse(fs.readFileSync(path.resolve(file)));
 
-console.log('Metatest CLI Stub');
+const splitOptions = v => v.split(',');
+
+const clOptions = [
+  [ '-c, --config <value>', 'Config file' ],
+  [ '-e, --enviroment <value>', 'Enviroment to run', 'node' ],
+  [ '-p, --required-packages <items>', 'Required packages', splitOptions, []],
+  [ '-b, --browsers <items>', 'Browsers to run', splitOptions, []],
+  [ '-p, --browser-port <n>', 'Browser port'],
+  [ '-l, --browser-log <value>', 'Browser log level']
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error': [ 'LOG_ERROR', 'errors-only' ],
+  'warn': [ 'LOG_WARN', 'minimal' ],
+  'info': [ 'LOG_INFO', 'normal' ],
+  'debug': [ 'LOG_DEBUG', 'verbose' ]
+};
+
+const browserLaunchers = {
+  'Chrome': 'karma-chrome-launcher',
+  'Firefox': 'karma-firefox-launcher',
+  'IE': 'karma-ie-launcher',
+  'Opera': 'karma-opera-launcher',
+  'PhantomJS': 'karma-phantomjs-launcher',
+  'Safari': 'karma-safari-launcher'
+};
+
+const rmDuplicates = arr => arr.reduce((acc, cur) => {
```

Comment:
Maybe `Array.from(new Set(arr))`? At least it's shorter)

Reply:
I don't like the idea of additional memory copying. What about performance? Is it about the same?

Reply:
I don't think it matters, this is not a hot code. The memory usage shouldn't matter much, there is an array of like 5 elements.
Though if you prefer you can leave it as is.